std/assembly/math.ts

```diff
@@ -1,4 +1,10 @@
+/// <reference path="./util/seedrandom.d.ts" />
+
+import { ArrayBufferView } from "./arraybuffer";
 import * as JSMath from "./bindings/Math";
+import * as JSDate from "./bindings/Date";
+import * as wasi from "./bindings/wasi_snapshot";
```

Comment:
There's something wrong here in that `"./bindings/wasi"` does not work. Can't find the symbols. Thought we had fixed those `export *` / `import *` problems :(